### DISCUSSION TITLE
Collaborators for private repositories with plan Pro

### DISCUSSION THREAD

**Discussion by KristofMerey** [community/community#44864] [New to GitHub]
The page [Compare plans](https://github.com/settings/billing/plans) has a row with "Collaborators for private repositories."  Under the plan Pro, we can see $4 per user/month here. What does it mean? Now, I have two private repos, in which I work with some friends. If I upgrade to Pro, do they lose their access unless they also upgrade to Pro? Or can we not use the extra features in these repos? Or do I have to pay an additional $4 for every contributor?

**Comment by queenofcorgis** [comment]
On the Pro plan users can have unlimited collaborators and it doesn't cost them $4/month extra. It only costs more per collaborator on private repos on the Team and Enterprise Cloud plan for organization accounts. We're working on updating our documentation to better reflect this 😄
